### PR TITLE
Jamf Protect Tile

### PR DESCRIPTION
**Pull request by apiazza-dd**
### What does this PR do?
This PR adds a tile for Jamf Protect

A brief description of the change being made with this pull request.

### Motivation
Jamf has created a push web integration for their Protect Product. The following PR contains the tile setup with configuration instructions. 

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by FlorentClarret — APPROVED** [review]

File: jamf_protect/manifest.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a863cfe8-5ba4-45ae-923c-d273510f099c",
+  "app_id": "jamf-protect",
+  "display_on_public_website": false,
```
Comment (FlorentClarret) [inline jamf_protect/manifest.json:5]:
```suggestion
  "display_on_public_website": true,
```

We use this field to display the integrations in the docs (for instance [this page](https://docs.datadoghq.com/integrations/kafka/?tab=host) for kafka)

**Review by ZacharyAnderson — APPROVED** [review]

File: jamf_protect/manifest.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "manifest_version": "2.0.0",
+  "app_uuid": "a863cfe8-5ba4-45ae-923c-d273510f099c",
+  "app_id": "jamf-protect",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Endpoint security & mobile threat defense (MTD) for Mac & mobile devices",
```
Comment (alai97) [inline jamf_protect/manifest.json:11]:
```suggestion
    "description": "Endpoint security and mobile threat defense (MTD) for Mac and mobile devices.",
```

File: jamf_protect/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        1.  **Header Name**: DD-APPLICATION-KEY
+    iv.  Click **Create option "DD-APPLICATION-KEY"**
+        1.  **Header Value:** <APPLICATION_KEY>
+    6.  Click **Test Configuration**
+    7.  If successful, click **Create Configuration**
+
+
+### Validation
+
+Navigate to Datadogs logs product and search for `source:jamf` to validate you're receiving logs.
```
Comment (alai97) [inline jamf_protect/README.md:103]:
```suggestion
Navigate to the [Logs Explorer][5] and search for `source:jamf` to validate you are receiving logs.
```

File: jamf_protect/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+## Data Collected
+[Jamf Audit Logs][1]
```
Comment (alai97) [inline jamf_protect/README.md:106]:
```suggestion

### Logs

The Jamf Protect integration allows you to send [Jamf Audit Logs][4] to Datadog.
```

File: jamf_protect/README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+Jamf Protect does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][3].
+
+[1]: https://www.jamf.com/products/jamf-protect/
+[2]: https://app.datadoghq.com/account/settings/agent/latest
+[3]: https://docs.datadoghq.com/help/
+[4]: https://learn.jamf.com/bundle/jamf-protect-documentation/page/Audit_Logs.html
```
Comment (alai97) [inline jamf_protect/README.md:127]:
```suggestion
[4]: https://learn.jamf.com/bundle/jamf-protect-documentation/page/Audit_Logs.html
[5]: https://app.datadoghq.com/logs
[6]: https://app.datadoghq.com/integrations
[7]: https://docs.datadoghq.com/api/latest/logs/#send-logs
[8]: https://docs.datadoghq.com/getting_started/site/
```

File: jamf_protect/README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
+# Jamf Protect
+
+## Overview
+
+This check monitors [Jamf Protect][1].
```
Comment (alai97) [inline jamf_protect/README.md:5]:
```suggestion
This check monitors [Jamf Protect][1], which enables endpoint security, threat defense, and content filtering for Apple Mobile and Mac.
```

File: jamf_protect/README.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+## Setup
+
+
+### Installation
+
+Navigate to the Integrations Page and search for the "Jamf Protect" tile.
```
Comment (alai97) [inline jamf_protect/README.md:12]:
```suggestion
Navigate to the [Integrations page][6] and search for the "Jamf Protect" tile. 
```

File: jamf_protect/README.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+## Setup
+
+
+### Installation
+
+Navigate to the Integrations Page and search for the "Jamf Protect" tile. 
+
+### Configuration
+**Prerequisite: Determine your Datadog Intake URL:**
+Using Datadog's [Logs Documentation](https://docs.datadoghq.com/api/latest/logs/#send-logs), determine what your intake URL is by selecting your Datadog Site in the top right corner. If you don't know what your Datadog Site is, please see [these docs](https://docs.datadoghq.com/getting_started/site/).**macOS Security Portal:**
```
Comment (alai97) [inline jamf_protect/README.md:16]:
```suggestion

### Determine your Datadog Intake URL

Using the [Datadog API Logs documentation][7], determine what your intake URL is by selecting your [Datadog Site][8] on the top right corner. 
```

File: jamf_protect/README.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+        i.  **Name**: DD-API-KEY
+        ii.  **Value**: <API_Key>
+    c.  Click **+ Add HTTP Header**
+        i.  **Name**: DD-APPLICATION-KEY
+        ii.  **Value**: <APPLICATION_KEY>
+
+7.  Click **Save**
+
+
+### Jamf Security Cloud:
```
Comment (alai97) [inline jamf_protect/README.md:54]:
```suggestion
### Jamf Security Cloud
```

File: jamf_protect/README.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+1.  Click **Actions**
+2.  Click **Create Actions**
+3.  **Name:** Datadog
+4.  Click **Remote Alert Collection Endpoints**
+    a.  **URL: **`https://${DATADOG_INTAKE_URL}/api/v2/logs?ddsource=jamfprotect&service=alerts`
+    b.  **Min Severity & Max Severity:** Set by you
+    c.  **Click + Add HTTP Header**
+        i.  Name: DD-API-KEY
+        ii.  Value: <API_Key>
+    d.  Click **+ Add HTTP Header**
```
Comment (alai97) [inline jamf_protect/README.md:29]:
```suggestion
1.  Click **Actions**.
2.  Click **Create Actions**.
3.  **Name:** Datadog.
4.  Click **Remote Alert Collection Endpoints**.
    a.  **URL:** `https://${DATADOG_INTAKE_URL}/api/v2/logs?ddsource=jamfprotect&service=alerts`
    b.  Set **Min Severity & Max Severity:**.
    c.  Click **+ Add HTTP Header**.
        i.  Name: DD-API-KEY
        ii.  Value: <API_Key>
    d.  Click **+ Add HTTP Header**.
```

File: jamf_protect/README.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+        i.  Name: DD-APPLICATION-KEY
+        ii.  Value: <APPLICATION_KEY>
+5.  Click **+ Unified Logs Collection Endpoints**
+    a.  **URL:** `https://${DATADOG_INTAKE_URL}/api/v2/logs?ddsource=jamfprotect&service=unifiedlogs`
+    b.  Click + **Add HTTP Header**
+            i.  **Name**: DD-API-KEY
+            ii.  **Value**: <API_Key>
+    c.  Click **+ Add HTTP Header**
+        i.  **Name**: DD-APPLICATION-KEY
+        ii.  **Value**: <APPLICATION_KEY>
```
Comment (alai97) [inline jamf_protect/README.md:40]:
```suggestion
5.  Click **+ Unified Logs Collection Endpoints**.
    a.  **URL:** `https://${DATADOG_INTAKE_URL}/api/v2/logs?ddsource=jamfprotect&service=unifiedlogs`
    b.  Click + **Add HTTP Header**.
            i.  **Name**: DD-API-KEY
            ii.  **Value**: <API_Key>
    c.  Click **+ Add HTTP Header**.
        i.  **Name**: DD-APPLICATION-KEY
        ii.  **Value**: <APPLICATION_KEY>
```

File: jamf_protect/README.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+6.  Click **+ Telemetry Collection Endpoints**
+    a.  **URL: **`https://${DATADOG_INTAKE_URL}/api/v2/logs?ddsource=jamfprotect&service=telemetry`
+    b.  Click** + Add HTTP Header**
+        i.  **Name**: DD-API-KEY
+        ii.  **Value**: <API_Key>
+    c.  Click **+ Add HTTP Header**
+        i.  **Name**: DD-APPLICATION-KEY
+        ii.  **Value**: <APPLICATION_KEY>
+
+7.  Click **Save**
```
Comment (alai97) [inline jamf_protect/README.md:51]:
```suggestion
6.  Click **+ Telemetry Collection Endpoints**.
    a.  **URL: **`https://${DATADOG_INTAKE_URL}/api/v2/logs?ddsource=jamfprotect&service=telemetry`
    b.  Click** + Add HTTP Header**.
        i.  **Name**: DD-API-KEY
        ii.  **Value**: <API_Key>
    c.  Click **+ Add HTTP Header**.
        i.  **Name**: DD-APPLICATION-KEY
        ii.  **Value**: <APPLICATION_KEY>

7.  Click **Save**.
```

File: jamf_protect/README.md [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+    e.  **Endpoint:** `api/v2/logs?ddsource=jamfprotect&service=threatevents`
+    f.  **Additional Headers:**
+        i.  **Header Name:** DD-API-KEY
+            1.  Click **Create option "DD-API-KEY"**
+    i.  **Header Value:** <API_Key>
+    ii.  **Header Name**: DD-APPLICATION-KEY
+        1.  Click **Create option "DD-APPLICATION-KEY"**
+    iii.  **Header Value:** <APPLICATION_KEY>
+        1.  Click **Test Configuration**
+    7.  If successful, click **Create Configuration**
```
Comment (alai97) [inline jamf_protect/README.md:76]:
```suggestion

1.  Click **Integrations** in the Threat Events Stream.
2.  Click **Data Streams**.
3.  Click **New Configuration**.
4.  Select **Threat Events**.
    a.  Select **Generic HTTP**.
5.  Click **Continue**.
    a.  **Configuration** **Name:** Datadog (Threat)
    b.  **Protocol:** **HTTPS**
    c.  **Server** **Hostname/IP:** `${DATADOG_INTAKE_URL}`
    d.  **Port:** **443**
    e.  **Endpoint:** `api/v2/logs?ddsource=jamfprotect&service=threatevents`
    f.  **Additional Headers:**
        i.  **Header Name:** DD-API-KEY
            1.  Click **Create option "DD-API-KEY"**.
    i.  **Header Value:** <API_Key>
    ii.  **Header Name**: DD-APPLICATION-KEY
        1.  Click **Create option "DD-APPLICATION-KEY"**.
    iii.  **Header Value:** <APPLICATION_KEY>
        1.  Click **Test Configuration**.
    7.  If successful, click **Create Configuration**.
```

File: jamf_protect/README.md [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+    e.  **Endpoint:** `api/v2/logs?ddsource=jamfprotect&service=networktraffic`
+6.  **Additional Headers:**
+    i.  **Header Name:** DD-API-KEY
+        1.  Click **Create option "DD-API-KEY"**
+    ii.  **Header Value:** <API_Key>
+        1.  **Header Name**: DD-APPLICATION-KEY
+    iv.  Click **Create option "DD-APPLICATION-KEY"**
+        1.  **Header Value:** <APPLICATION_KEY>
+    6.  Click **Test Configuration**
+    7.  If successful, click **Create Configuration**
```
Comment (alai97) [inline jamf_protect/README.md:98]:
```suggestion
1.  Click **Integrations**.
2.  Click **Data Streams**.
3.  Click **New Configuration**.
4.  Select **Threat Events**.
    a.  Select **Generic HTTP**.
5.  Click **Continue**.
    a.  **Configuration** **Name:** Datadog (Threat)
    b.  **Protocol:** **HTTPS**
    c.  **Server** **Hostname/IP:** `${DATADOG_INTAKE_URL}`
    d.  **Port:** **443**
    e.  **Endpoint:** `api/v2/logs?ddsource=jamfprotect&service=networktraffic`
6.  **Additional Headers:**
    i.  **Header Name:** DD-API-KEY
        1.  Click **Create option "DD-API-KEY"**.
    ii.  **Header Value:** <API_Key>
        1.  **Header Name**: DD-APPLICATION-KEY
    iv.  Click **Create option "DD-APPLICATION-KEY"**.
        1.  **Header Value:** <APPLICATION_KEY>
    6.  Click **Test Configuration**.
    7.  If successful, click **Create Configuration**.
```

**Review by alai97 — COMMENTED** [review]
Some copy suggestions!

**Review by alai97 — APPROVED** [review]
Looks good!

**Pull request merged**